CMakeLists.txt

```diff
@@ -655,6 +656,26 @@ IF (USE_TESTS)
 	LIST(APPEND CMAKE_CTEST_ARGUMENTS "--output-on-failure")
 ENDIF()
 
+IF(USE_TRACY)
+	INCLUDE(FetchContent)
+
+	IF (NOT STATIC_LINK)
+		SET(CMAKE_POSITION_INDEPENDENT_CODE ON)
+	ENDIF()
+
+	FETCHCONTENT_DECLARE(
+		tracy
+		GIT_REPOSITORY https://github.com/wolfpld/tracy.git
+		GIT_TAG v0.10
+		GIT_SHALLOW TRUE
+		GIT_PROGRESS TRUE
+	)
+	FETCHCONTENT_MAKEAVAILABLE(tracy)
+
+	ADD_DEFINITIONS("-DUSE_TRACY")
+	INCLUDE_DIRECTORIES(${tracy_SOURCE_DIR}/public)
```

Comment:
FYI, this should end up in `${CMAKE_BINARY_DIR}/_deps` meaning we don't need to add an exclusion for `make dist` purposes.

Reply:
`${tracy_SOURCE_DIR}` is `build/_deps/tracy-src`, so there is not interference (I just run `make dist`) or I don't get what you mean.

Reply:
Just documented that nothing more needs to be done.